### DISCUSSION TITLE
TileJSON: Request with 'https', but returns 'http'

### DISCUSSION THREAD

**Discussion by huanglii** [developmentseed/titiler#969] [Q&A] [answered]
#### Problem description

[this should explain **why** the current behavior is a problem and why the expected output is a better solution.]

When I request a tilejson with 'https', it returns the tiles with 'http'.

https://example.com/titiler/cog/tilejson.json?url=...

```json
{
    "tiles": [
        "http://example.com/titiler/cog/tiles/WebMercatorQuad/{z}/{x}/{y}@1x?url=..."
    ]
}
```

#### Expected Output

```json
{
    "tiles": [
        "https//example.com/titiler/cog/tiles/WebMercatorQuad/{z}/{x}/{y}@1x?url=..."
    ]
}
```

#### Environment Information

Docker imges: ghcr.io/developmentseed/titiler:0.18.5

**Comment by vincentsarago** [comment, marked as answer]
This is mostly due to a bad proxy configuration on your deployment 

https://github.com/developmentseed/titiler/discussions/345 https://github.com/developmentseed/titiler/discussions/342
